cegs_portal/templates/base.html

```diff
@@ -1,82 +1,155 @@
+                                </svg>
+                            </button>
+                        </div>
+                    </div>
 
-        <div class="collapse navbar-collapse" id="navbarSupportedContent">
-          <ul class="navbar-nav mr-auto">
-            <li class="nav-item active">
-              <a class="nav-link" href="{% url 'home' %}">Home <span class="sr-only">(current)</span></a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="{% url 'about' %}">About</a>
-            </li>
-            {% if request.user.is_authenticated %}
-              <li class="nav-item">
-                {# URL provided by django-allauth/account/urls.py #}
-                <a class="nav-link" href="{% url 'users:detail' request.user.username  %}">{% translate "My Profile" %}</a>
-              </li>
-              <li class="nav-item">
-                {# URL provided by django-allauth/account/urls.py #}
-                <a class="nav-link" href="{% url 'account_logout' %}">{% translate "Sign Out" %}</a>
-              </li>
-            {% else %}
-              <li class="nav-item">
-                {# URL provided by django-allauth/account/urls.py #}
-                <a id="sign-up-link" class="nav-link" href="{% url 'account_signup' %}">{% translate "Sign Up" %}</a>
-              </li>
-              <li class="nav-item">
-                {# URL provided by django-allauth/account/urls.py #}
-                <a id="log-in-link" class="nav-link" href="{% url 'account_login' %}">{% translate "Sign In" %}</a>
-              </li>
-            {% endif %}
-          </ul>
-        </div>
-      </nav>
+                    <div class="menu w-full lg:block flex-grow lg:flex lg:items-center lg:w-auto lg:px-3 px-8">
+                        <div class="text-md font-bold lg:flex-grow">
+                            <a
+                                href="{% url 'home' %}"
+                                class="block mt-4 lg:inline-block lg:mt-0 text-white hover:text-white px-2 py-2 rounded hover:bg-blue-700 no-underline"
+                                style="color: white;"
+                                >
+                                <i class="bi bi-house-door"></i> Home
+                            </a>
+                            <a
+                                href="{% url 'about' %}"
+                                class="block mt-4 lg:inline-block lg:mt-0 text-white hover:text-white px-2 py-2 rounded hover:bg-blue-700 no-underline"
+                                style="color: white;"
+                                >
+                                <i class="bi bi-patch-question"></i> About
+                            </a>
+                        </div>
+                        <!-- This is search component -->
+                        <div id="hide-nav-searchbar" class="relative mx-auto text-gray-600 lg:block hidden">
```

Comment:
The id should be descriptive of the item it's attached to, like "nav-searchbar".